strax/testutils.py

```diff
@@ -260,6 +260,74 @@ def compute(self, peaks):
         return dict(peak_classification=p,
                     lone_hits=lh)
 
+        
+# Plugins with time structure within chunks,
```

Comment:
Am I right that this is identical to Records from testutils, apart from setting the record times differently? Could we inherit from that one (maybe after modifying it) to reduce duplication? 

(Maybe this is not possible of course; I vaguely recall something about inheritance messing up config options.)